src/ResourceManager/Profile/Commands.Profile/Environment/AddAzureRMEnvironment.cs

```diff
@@ -112,12 +121,31 @@ protected override void BeginProcessing()
         public override void ExecuteCmdlet()
         {
             var profileClient = new RMProfileClient(AzureRmProfileProvider.Instance.Profile);
+            var result = profileClient.ListEnvironments(Name).Select(s => (PSAzureEnvironment)s).ToList();
```

Comment:
we don't need this check. its a breaking change. Add-AzureRmEnvironment providing the same name will update the existing properties.. we should keep that behavious as is